README.md

```diff
@@ -199,15 +199,17 @@ This type of button component should be used with simple icon-only or text-only
 
 ## Controlling gesture handlers interactions
 
-Gesture handler library API allows for defining some basic interaction between handler components. Interactions can be defined by first setting a string identifer for a handler component with the `id` property and then referencing it with `waitFor` or `simultaneousHandlers` props in other handler component.
+Gesture handler library API allows for defining some basic interaction between handler components. Interactions can be defined by first setting a ref object (`React.createRef()`) for a handler component with the `ref` property and then referencing it with `waitFor` or `simultaneousHandlers` props in other handler component.
```

Comment:
maybe link to https://reactjs.org/docs/refs-and-the-dom.html so that it is clear that it the standard way of setting refs as of React 16.3